src/lib/queryAdvise.js

```diff
@@ -111,7 +111,11 @@ export function getAllPossibileQueries(element) {
     .filter((query) => query.type !== 'GENERIC')
     .map((query) => {
       const method = getFieldName(query.method);
-      return getSuggestedQuery(element, 'get', method);
+      try {
+        return getSuggestedQuery(element, 'get', method);
+      } catch (e) {
+        return undefined;
+      }
```

Comment:
I'm not a big fan of having a `try/catch` inside a loop. Also, shouldn't this be fixed upstream?

Reply:
Because if other queries don't throw an exception I want to show them

Reply:
I understand. And isn't this fixable in @testing-library/dom?

Reply:
This problem is that it is a new usecase.

We can solve the problem anyway but I think is better to maintain this code to avoid other problem of this kind